htdocs/fourn/class/fournisseur.facture-rec.class.php

```diff
@@ -0,0 +1,2209 @@
+                $this->user_modif               = $obj->fk_user_modif;
+                $this->fk_project               = $obj->fk_projet;
+                $this->fk_account               = $obj->fk_account;
+                $this->mode_reglement_id        = $obj->fk_mode_reglement;
+                $this->mode_reglement_code      = $obj->mode_reglement_code;
+                $this->mode_reglement           = $obj->mode_reglement_libelle;
+                $this->cond_reglement_id        = $obj->fk_cond_reglement;
+                $this->cond_reglement_code      = $obj->cond_reglement_code;
+                $this->cond_reglement           = $obj->cond_reglement_libelle;
+                $this->cond_reglement_doc       = $obj->cond_reglement_libelle_doc;
+                $this->date_lim_reglement       = $this->db->jdate($obj->date_lim_reglement);
+                $this->note_private             = $obj->note_private;
+                $this->note_public              = $obj->note_public;
+                $this->model_pdf                = $obj->modelpdf;
+
+                // Multicurrency
+                $this->fk_multicurrency 		= $obj->fk_multicurrency;
+                $this->multicurrency_code       = $obj->multicurrency_code;
+                $this->multicurrency_tx 		= $obj->multicurrency_tx;
+                $this->multicurrency_total_ht   = $obj->multicurrency_total_ht;
+                $this->multicurrency_total_tva 	= $obj->multicurrency_total_tva;
+                $this->multicurrency_total_ttc 	= $obj->multicurrency_total_ttc;
+
+                $this->usenewprice			    = $obj->usenewprice;
+                $this->frequency			    = $obj->frequency;
+                $this->unit_frequency           = $obj->unit_frequency;
+                $this->date_when			    = $this->db->jdate($obj->date_when);
+                $this->date_last_gen            = $this->db->jdate($obj->date_last_gen);
+                $this->nb_gen_done			    = $obj->nb_gen_done;
+                $this->nb_gen_max               = $obj->nb_gen_max;
+                $this->auto_validate            = $obj->auto_validate;
+                $this->generate_pdf             = $obj->generate_pdf;
+
+
+                if ($this->statut == self::STATUS_DRAFT) {
+                    $this->brouillon = 1;
+                }
+
+                // Retrieve all extrafield
+                // fetch optionals attributes and labels
+                $this->fetch_optionals();
+
+                /*
+                 * Lines
+                 */
+                $result = $this->fetch_lines();
+                if ($result < 0) {
+                    $this->error = $this->db->lasterror();
+                    return -3;
+                }
+                return 1;
+            } else {
+                $this->error = 'Bill with id '.$rowid.' or ref '.$ref.' not found';
+                dol_syslog('Facture::Fetch Error '.$this->error, LOG_ERR);
```

Comment:
On est pas tellement dans la methode fetch de la classe Facture :smirk: 
```suggestion
                dol_syslog(__METHOD__.' Error '.$this->error, LOG_ERR);
```

Reply:
Done